nilearn/decomposition/multi_pca.py

```diff
@@ -211,7 +221,7 @@ def fit(self, imgs, y=None, confounds=None):
             raise ValueError('Need one or more Niimg-like objects as input, '
                              'an empty list was given.')
         if confounds is None:
-            confounds = itertools.repeat(None, len(imgs))
+            confounds = [None] * len(imgs)  # itertools.repeat(None, len(imgs))
```

Comment:
These lines are found in both MultiPCA.fit() and .transform(). Chance both the same way for consistency? Also we could loose the itertools-import then.